packages/@markuplint/rules/src/attr-duplication/index.ts

```diff
@@ -4,10 +4,10 @@ export default createRule({
 	name: 'attr-duplication',
 	defaultValue: null,
 	defaultOptions: null,
-	async verify(document, translate) {
+	verify(document, translate) {
```

Comment:
@JounQin 

Removing `async/await` keywords are not needed yet.
It is another story.

Now important, the run correctly whatever the case sync/async.
You should create new tests of the cases.
Create dummy custom rules, and test through that.

Reply:
@YusukeHirao 

Hi, I mentioned that `rule-textlint` test cases already prove that it runs well with `async` rules. If you need more test cases, I'll add more. For example, mixing `aync` and `sync` rules.

Reply:
@JounQin 

I understand you want to prove to pass the current tests.
But I said, "Don't change tests".

Please see to me you prove that through "new tests".

Reply:
OK, I'll add more test cases. 😜

Reply:
@YusukeHirao 

Test cases added!

Reply:
@JounQin 

Revert removing keywords.

Reply:
You mean `async/await` in rules? They must be removed to support sync mode.

Reply:
@JounQin 

What?! I don't hope for a breaking change.
I don't accept if It cannot support async mode.

I thought this PR is supporting both sync mode and async mode.

Reply:
@YusukeHirao 

This PR support both sync and async...

I just change all rules to be runnable in sync mode.

`rule-textlint` is fully async.

Reply:
`sync` rules are always supported in `async` mode, so there should be no breaking change for end users.

Reply:
Well if so, it doesn't need to change that.
I will decide whether to remove async/await keywords.
Again, change only the minimum required.

Reply:
OK, wait a second.

Reply:
@YusukeHirao Done!

Reply:
@JounQin 

Great! I WANTED THIS!